### DISCUSSION TITLE
第5节课作业(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#230] [作业]
请提交作业的链接 Please submit a link to the assignment.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/679241631

**Comment by jayjayhust** [comment]
https://zhuanlan.zhihu.com/p/679618970

**Comment by water2silver** [comment]
课程笔记与课程作业
https://zhuanlan.zhihu.com/p/679757890

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/679809908

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/679783071

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/LMDeploy-3773d01f0d14485cb2dff660c851834d

**Comment by YongXie66** [comment]
笔记与作业
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/Lesson5.pdf

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/5
